docs/components/plugins/api/APISectionMethods.tsx

```diff
@@ -118,13 +118,15 @@ export const APIMethod = ({
   isReturnTypeReference = false,
   exposeInSidebar = false,
   parameters = [],
+  platforms = [],
 }: {
   exposeInSidebar?: boolean;
   name: string;
   comment: string;
   returnTypeName: string;
   isProperty: boolean;
   isReturnTypeReference: boolean;
+  platforms: ('Android' | 'iOS' | 'Web')[];
```

Comment:
```suggestion
  platforms: ('Android' | 'iOS' | 'Web')[];
```

We should probably stick to lowercase versions and lowercase the array content on compare, so casing doesn't matter. 

Also, we have few places already passes platforms as prams, so in the future it would be nice to create an universal type for this, but it is out-of-scope of this PR I think.

Reply:
We should stick to lowercase versions, but it requires major changes in other parts of the API. I don't want to touch them right now.